### ISSUE TITLE
Documentation needs update

### ISSUE THREAD

**Issue by PerfectlyFineCode** [AvaloniaCommunity/Material.Avalonia#179]
Documentation needs updating

```cs
Material.Styles.Themes.MaterialTheme
```
is not available anymore.
```cs
Material.Styles.Themes.BundledTheme
```
However is available, but there's no documentation on how to use it.

**Comment by SKProCH** [comment]
Hello, @PerfectlyFineCode 

`MaterialTheme` available in a pre release version - 3.0.0 and nightly builds.
`BundledTheme` described [here](https://github.com/AvaloniaCommunity/Material.Avalonia/wiki/Simple-getting-started):
![image](https://user-images.githubusercontent.com/29896317/181908334-d135b211-cd8e-44c3-9d53-5a9de0f9ded6.png)

But docs really need some changes, yep.

**Comment by SKProCH** [comment]
I rewrote `Getting started` page, it looks better now.
